### PR TITLE
[US435530]: Persist changes made to release dockerized version of UAA 4.30.0

### PR DESCRIPTION
**Pull request by emarmol**
**Story**: [US435530](https://rally1.rallydev.com/#/detail/userstory/410715018988/tasks?fdp=true): Persist changes made to release dockerized version of UAA 4.30.0 for future release

**Task**: Persist using build.ge artifactory and predix-virtual in predix_prunes/jwt_grant_4.30.0

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -60,33 +60,18 @@ allprojects {
...
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/MAAXA-MVN-SNAPSHOT'
-        }
-        maven {
-            url 'https://build.shibboleth.net/nexus/content/repositories/releases/'
+            url 'https://artifactory.build.ge.com/artifactory/MAAXA'
         }
+        maven { url 'https://artifacts.alfresco.com/nexus/content/repositories/public/' }
+        maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases/' }
```
Comment (ylongva) [inline build.gradle:74]: "build.shibboleth.net" is not reachable. It was removed in this commit https://github.com/GESoftware-CF/uaa/commit/74c1ec27e004ce4d3822c99b3637204eac00c527. Should we also include the commit in this PR?

Comment (irinaepshteyn) [reply build.gradle:74]: Good catch.  This branch should never be built from Jenkins. However, to be consistent and make merging of this branch into predix_extensions in the future easier, it makes sense to pick up this commit as well.

File: build.gradle [hunk 2]
```diff
@@ -60,33 +60,18 @@ allprojects {
...
-        maven {
-            credentials {
-                username geArtifactoryReader
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/MAAXA-MVN-SNAPSHOT'
-        }
-        maven {
-            url 'https://build.shibboleth.net/nexus/content/repositories/releases/'
+            url 'https://artifactory.build.ge.com/artifactory/MAAXA'
```
Comment (irinaepshteyn) [inline build.gradle:71]: MAAXA repo is not required now.  jwt-bearer-key-provider is available in predix-virtual now.  We could remove lines 65-72.

**Comment by irinaepshteyn** [comment]
As you notice there is no hook to build this PR from Jenkins.  It's because this branch is never intended to be built by itself there.

File: build.gradle (hunk 1 above)
Comment (emarmol) [reply build.gradle]: Thanks @ylongva & @irinaepshteyn, I cherry picked that commit as well in this PR

File: build.gradle (hunk 2 above)
Comment (emarmol) [reply build.gradle]: This was fixed with the last commit

**Pull request merged**